### ISSUE TITLE
[Sprint] Enable using different learning rate per modules and add option for no weight decay on bias and norm weight

### ISSUE THREAD

**Issue by illian01** [Nota-NetsPresso/netspresso-trainer#528]
## Description

- Learning rate can be applied differently according to the modules (backbone, neck, head)
- weigh decay can be 0 for bias and norm weight

## Checklists

- [ ] I would create a corresponding branch for this issue from the designated (mostly `dev`) branch.
- [x] This issue only contains a preceding agreement between project owners.
